### ISSUE TITLE
[Bug] Unable to get application environment property

### ISSUE THREAD

**Issue by julianpoemp** [buttercup/buttercup-desktop#1055]
I upgraded to Buttercup 2.8.0 and tried to add a new entry. After it had not worked (see #1053) I tried to edit an entry and add a new custom field. Then I got this error message:

````
Error: Unable to get application environment property: 'rng/v1/uuid' does not exist
    at De (file:///Applications/Buttercup.app/Contents/Resources/app.asar/build/renderer/index.js:215:6335)
    at Ce (file:///Applications/Buttercup.app/Contents/Resources/app.asar/build/renderer/index.js:525:1531862)
    at t.VaultEditor (file:///Applications/Buttercup.app/Contents/Resources/app.asar/build/renderer/index.js:628:51)
    at h (file:///Applications/Buttercup.app/Contents/Resources/app.asar/build/renderer/index.js:588:45)
    at div
    at styled.div (file:///Applications/Buttercup.app/Contents/Resources/app.asar/build/renderer/index.js:525:1533766)
    at t.SearchProvider (file:///Applications/Buttercup.app/Contents/Resources/app.asar/build/renderer/index.js:697:2527)
    at div
    at styled.div (file:///Applications/Buttercup.app/Contents/Resources/app.asar/build/renderer/index.js:525:1533766)
    at t.VaultManagement (file:///Applications/Buttercup.app/Contents/Resources/app.asar/build/renderer/index.js:638:51)
    at t (file:///Applications/Buttercup.app/Contents/Resources/app.asar/build/renderer/index.js:525:1436902)
    at t (file:///Applications/Buttercup.app/Contents/Resources/app.asar/build/renderer/index.js:525:1438871)
    at t (file:///Applications/Buttercup.app/Contents/Resources/app.asar/build/renderer/index.js:525:1433509)
    at t (file:///Applications/Buttercup.app/Contents/Resources/app.asar/build/renderer/index.js:525:1440271)
    at div
    at styled.div (file:///Applications/Buttercup.app/Contents/Resources/app.asar/build/renderer/index.js:525:1533766)
    at Ce (file:///Applications/Buttercup.app/Contents/Resources/app.asar/build/renderer/index.js:525:1531862)
    at t.App (file:///Applications/Buttercup.app/Contents/Resources/app.asar/build/renderer/index.js:530:30)
````

### System Information
MacOS 11.4
Buttercup 2.8.0
Connection via WebDAV from Owncloud

**Comment by stanschus** [comment]
System Info:
Ubuntu 20.04
Buttercup 2.8.0
AppImage
local Buttercup Vault
local Vaults were create via deb package (buttercup 1.20.5)

editing an existing entry gives following error message:

``A fatal error has occurred - we're sorry this has happened. Please check out the details below in case they help diagnose the issue:

Error: Unable to get application environment property: 'rng/v1/uuid' does not exist
    at De (file:///tmp/.mount_Butter0zkpqr/resources/app.asar/build/renderer/index.js:215:6335)
    at Ce (file:///tmp/.mount_Butter0zkpqr/resources/app.asar/build/renderer/index.js:525:1531862)
    at t.VaultEditor (file:///tmp/.mount_Butter0zkpqr/resources/app.asar/build/renderer/index.js:628:51)
    at h (file:///tmp/.mount_Butter0zkpqr/resources/app.asar/build/renderer/index.js:588:45)
    at div
    at styled.div (file:///tmp/.mount_Butter0zkpqr/resources/app.asar/build/renderer/index.js:525:1533766)
    at t.SearchProvider (file:///tmp/.mount_Butter0zkpqr/resources/app.asar/build/renderer/index.js:697:2527)
    at div
    at styled.div (file:///tmp/.mount_Butter0zkpqr/resources/app.asar/build/renderer/index.js:525:1533766)
    at t.VaultManagement (file:///tmp/.mount_Butter0zkpqr/resources/app.asar/build/renderer/index.js:638:51)
    at t (file:///tmp/.mount_Butter0zkpqr/resources/app.asar/build/renderer/index.js:525:1436902)
    at t (file:///tmp/.mount_Butter0zkpqr/resources/app.asar/build/renderer/index.js:525:1438871)
    at t (file:///tmp/.mount_Butter0zkpqr/resources/app.asar/build/renderer/index.js:525:1433509)
    at t (file:///tmp/.mount_Butter0zkpqr/resources/app.asar/build/renderer/index.js:525:1440271)
    at div
    at styled.div (file:///tmp/.mount_Butter0zkpqr/resources/app.asar/build/renderer/index.js:525:1533766)
    at Ce (file:///tmp/.mount_Butter0zkpqr/resources/app.asar/build/renderer/index.js:525:1531862)
    at t.App (file:///tmp/.mount_Butter0zkpqr/resources/app.asar/build/renderer/index.js:530:30)``

**Comment by perry-mitchell** [comment]
Found the issue - the environment was not being initialised properly in the renderer.. hence the error. Don't know why this didn't appear in earlier versions as this requirement has always been there.

Releasing a patch now.

**Comment by perry-mitchell** [comment]
Released in 2.8.1
